feat: added aave <> immunefi activation AIP (#308)
bgd-labs/aave-proposals: src/20230920_AaveV3_Eth_AaveImmunefiActivation/AaveImmunefiActivation.md

---
title: "Aave <> Immunefi program activation"
author: "BGD Labs @bgdlabs"
discussions: "https://governance.aave.com/t/bgd-aave-immunefi-bug-bounty-program/14757"
---

## Simple Summary

This proposal approves the activation of the Aave <> Immunefi bug bounty program.

All the details about the program can be found on the associated governance forum post, and on the draft document [HERE](https://docs.google.com/document/d/1cst5Kx0N2t4PfyNAEpobH6dWUE2JKfo49dSjRMhelfY).

## Motivation

Having a continuous and updated bug bounty program is a really important component regarding security procedures of an ecosystem like Aave.

The [Immunefi platform](https://immunefi.com/) is a leader in the bug bounty space, adopted by multiple other blockchain protocols, and with a quite active network of security researches submitting reports.

Following the pre-approval on Snapshot, and within the context of our engagement, BGD has been working with Immunefi representatives to define and activate the program, respecting all the decentralized procedures of the Aave DAO, while keeping the maximum quality possible.

## Specification

> This proposal doesn't include any executable code, as it will only serve as binding agreement from the Aave DAO with Immunefi, to formally activated the bounty program.

Reviewer: ```suggestion
This proposal doesn't include any executable code, as it will only serve as a binding agreement between the Aave DAO and Immunefi, to formally activate the bounty program.
```
